finalize/finalize.gm9

```diff
@@ -23,8 +23,19 @@ if find "0:/Nintendo 3DS/$[SYSID0]/????????????????????????????????_user-id1" NU
 	set PREVIEW_MODE "0:/finalize/img/error18.png"
 	echo "Error #18: MSET9 detected\n \nMSET9 is still installed on this console.\nYou MUST remove it before continuing.\nScan the QR code above for instructions.\n \nIf this error persists, ask for help\non Discord: https://discord.gg/MWxPgEp"
 	poweroff
+
+# What is the NULL parameter here for? If I changed it, would it become a variable holding the name for the found folder?
+# Where can I read up on gm9script?
+# It was here https://github.com/eiiiiix/EixMode9/blob/master/HelloScript.gm9
```

Comment:
should use [upstream's helloscript](https://github.com/d0k3/GodMode9/blob/master/resources/sample/HelloScript.gm9) rather than random fork (content should be identical)